easybuild/easyblocks/generic/cmakeninja.py

```diff
@@ -38,9 +38,13 @@ class CMakeNinja(CMakeMake, MesonNinja):
 
     @staticmethod
     def extra_options(extra_vars=None):
-        """Define extra easyconfig parameters specific to CMakeMake."""
+        """Define extra easyconfig parameters specific to CMakeNinja."""
         extra_vars = CMakeMake.extra_options(extra_vars)
         extra_vars['generator'][0] = 'Ninja'
+        extra_vars.update({
+            key: value for key, value in MesonNinja.extra_options().items()
+            if key.startswith('build_') or key.startswith('install_')
```

Comment:
Why only these keys, and not everything custom that `MesonNinja` defines?

Reply:
Because we use it only for build&install. By now it defines:
- build_dir - Misleadingly named: This is the "Source dir", only used for Meson configure -> Issue for this?
- build_cmd - Should overwrite ConfigureMake
- configure_cmd - Also in CMakeMake, must NOT overwrite
- install_cmd - should overwrite ConfigureMake
- separate_build_dir - Also defined by CMakeMake, same semantic

So actually we only want `build_cmd` and `install_cmd` but I wanted to be at least slightly more general as the description of CMakeNinja is
> Support for configuring with CMake, **building** and **installing** with MesonNinja